### PR TITLE
kubernetes: Drop kubevirt support

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

**Comment by martinpitt** [comment]
Thanks! When merging, or force-pushing,  please add a reference to https://bugzilla.redhat.com/show_bug.cgi?id=1629608 to the commit log.

File: test/verify/check-openshift [hunk 1]
```diff
@@ -391,542 +391,6 @@ LABEL io.projectatomic.nulecule.atomicappversion="0.1.11" \
         b.click("#kubernetes-reconnect")
         b.wait_present("#service-list")
 
-
-class TestOpenshiftPrerelease(TestOpenshift):
-    provision = {
-        "machine1": { "address": "10.111.113.1/20" },
-        "openshift": { "image": "openshift-prerelease", "forward": { 30000: 30000, 8443: 8443 } }
-    }
```
Comment (martinpitt) [inline test/verify/check-openshift]: This part needs to stay - we still want to run the non-kubevirt TestOpenshiftPrerelease tests.

Comment (Gundersanne) [reply test/verify/check-openshift]: Oh totally missed that TestOpenshiftPrerelease inherited TestOpenshift, now I understand where those tests come from :D

**Comment by martinpitt** [comment]
FAIL: tools/test-static-code 2 pyflakes test

Please run the test manually. I can't see the actual error in the semaphore log, but I suppose it's an unused import or so.

**Comment by martinpitt** [comment]
Between semaphore builds [5](https://semaphoreci.com/cockpit-project/cockpit/branches/pull-request-10085/builds/5) and [6](https://semaphoreci.com/cockpit-project/cockpit/branches/pull-request-10085/builds/6) all were green, so good enough. Thanks!

**Pull request merged**